### PR TITLE
created support and mobile pages

### PR DESCRIPTION
**Pull request by 123brandon**
I made pages for support and mobile.
The css for the images on both pages is very wonky.
I wrote the list.html for mobile from scratch so the code layout is also not pretty.
:/

### CODE REVIEW

File: site/layouts/_default/single.html [hunk 1]
```diff
@@ -1,7 +1,11 @@
 {{ define "main" }}
 
     {{ partial "jumbotron" (dict  "title" .Title "subtitle" .Params.subtitle) }}
-    <div class="cms ph3 mw7 center">
+<<<<<<< Updated upstream
```
Comment (statik) [inline site/layouts/_default/single.html]: this is one of the trickier things about git, there was a merge conflict and these fishbone/conflict markers need to get removed. We can discuss on Zoom and I can help with resolving it.

File: site/layouts/mobile/list.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+                    </ul>
+        
+        <div class="flex flex-wrap mhn1">
+            <div class="w-100 w-50-ns ph1-ns">
+                <a href="https://apps.apple.com/app/id1146472597">
+                    <img alt="Download from the app store" src="/img/app-store-download.svg">
+                </a>
+            </div>
+            <div class="w-100 w-50-ns ph1-ns">
+                <a href="https://play.google.com/store/apps/details?id=com.enlist&utm_source=website&pcampaignid=MKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1">
```
Comment (statik) [inline site/layouts/mobile/list.html]: I think these images will need some sizes specified to get them to look consistent.

File: site/layouts/mobile/list.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+                        <li>&check;Find what needs your attention quickly</li>
+                        <li>&check;Search for the candidates you want</li>
+                        <li>&check;Access all your information anytime, anywhere</li>
+                        <li>&check;Collaborate with your team on the go</li>
+                    </ul>
+        
+        <div class="flex flex-wrap mhn1">
+            <div class="fl w-50-ns pl7">
+                <a href="https://apps.apple.com/app/id1146472597">
+                    <img alt="Download from the app store" src="/img/app-store-download.svg" style="width:220px;height:100px;">
```
Comment (knmurphy) [inline site/layouts/mobile/list.html]: I suggest removing the `style` parameter from the `img` element and instead adding `class="mw-100"` to the `img' element.

**Comment by statik** [comment]
great work, lets get this merged and iterate more in future branches!

**Pull request merged**